app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/MainActivity.java

```diff
@@ -49,6 +52,17 @@ private void checkIfLoggedIn() {
         }
     }
 
+    public void logOut(){
+       SharedPreferences loginData = PreferenceManager.getDefaultSharedPreferences(this);
```

Comment:
@zsofinnemeth you already have a sharedpreferences reference, you don't need another.